src/components/List/List.jsx

```diff
@@ -1,35 +1,74 @@
-import React, { useRef, forwardRef, useMemo } from "react";
+import React, { useRef, forwardRef, useMemo, useState, useCallback } from "react";
 import PropTypes from "prop-types";
 import cx from "classnames";
 import useMergeRefs from "../../hooks/useMergeRefs";
 import { VirtualizedListItems } from "components/List/VirtualizedListItems/VirtualizedListItems";
 import "./List.scss";
+import { keyCodes } from "../../constants/KeyCodes";
 
 const List = forwardRef(
   ({ className, id, component, children, dense, ariaLabel, ariaDescribedBy, renderOnlyVisibleItems, style }, ref) => {
     const componentRef = useRef(null);
+    const [focusIndex, setFocusIndex] = useState(0);
     const mergedRef = useMergeRefs({ refs: [ref, componentRef] });
     const Component = component;
+    const childrenRefs = useRef([]);
+    const changeFocus = focusIndex => {
```

Comment:
We prefer not to create anonymous functions inside our component render function if we can prevent this. I wrote suggestion for how to change the code for deleting this function.

Reply:
(The reason for this it that writing an anonymous function in a render method will create new function instance any render)